tests/common/src/main/resources/SparkSqlTestCode.txt

```diff
@@ -40,4 +40,27 @@ val sparkTable = spark.range(1000000).selectExpr("id", "concat('sym', cast((id %
 sparkTable.createOrReplaceTempView("sparkTable")
 spark.sql("select sym, avg(id) from sparkTable group by sym").collect()
 
+// Client Pool Driver Test - Start
+val connectionProperties = new Properties()
+connectionProperties.setProperty("pool-maxTotal", "50")
+connectionProperties.setProperty("pool-maxIdle", "20")
+connectionProperties.setProperty("pool-initialSize", "50")
+connectionProperties.setProperty("pool-minActive", "50")
+connectionProperties.setProperty("pool-minIdleSize", "10")
+connectionProperties.setProperty("driver", "io.snappydata.jdbc.ClientPoolDriver")
+val df = spark.read.jdbc(JDBC_URL, "ROWTABLE", connectionProperties)
+val result = df.select("COL1").groupBy("COL2").avg("COL1")
```

Comment:
Better to add some result checks so that we know in case there are incorrect results. Like:
```scala
val result = df.select("COL1").groupBy("COL2").avg("COL1").collect()
assert(result.length == 5) // expect
assert(result.toSeq.map(_.getDouble(0)).sorted == Seq(...)) // expected result 
```
Similarly for other queries.